.github/workflows/deploy-with-docker.yml

```diff
@@ -25,6 +25,17 @@ jobs:
         with:
           submodules: recursive
 
+      - uses: docker/setup-buildx-action@v1
```

Comment:
It seems that the introduction of this part leads to the pull of the just-built `dashboard:ci` ?

Reply:
@nic-chen thank you!! I find that buildx will not load built images to `docker images`. I'm trying to fix it!